app/models/collection.rb

```diff
@@ -1,6 +1,7 @@
-# Generated by curation_concerns:models:install
+# Generated by hyrax:models
```

Comment:
Technically this was generated by curation_concerns (before we moved to hyrax). But I don't feel strongly about this.